src/OrbitGl/App.cpp

```diff
@@ -468,7 +468,7 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
         ORBIT_CHECK(capture_started_callback_ != nullptr);
         capture_started_callback_(file_path);
 
-        if (!GetCaptureData().instrumented_functions().empty()) {
+        if (!GetCaptureData().GetAllProvidedScopeIds().empty()) {
```

Comment:
I'm not sure what this chunk of code is about. It was supposed to select Live tab if there's some dynamic instrumentation. But now we do so if there's D, MS or MA. So you've fixed a bug that we have never noticed. Right?

Reply:
Logically, yes but in practice it should act the same as before; this is called on OnCaptureStarted at which point there might be dynamically instrumented hooked functions, but we don't know if there are MS or MA functions until after the capture starts when it starts adding them, so I think it acts exactly the same as before (and it wasn't a bug before).

Reply:
Makes sense. Thanks.